src/coreclr/tests/src/GC/API/GC/AddUsageTest.cs

```diff
@@ -1,112 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
We also have `RemoveUsageTest` in this directory that has very similar structure. Delete it too?